sdk/tests/client/transaction_builder/basic_outputs.rs

```diff
@@ -2575,12 +2575,13 @@ fn automatic_allotment_provided_in_and_output() {
 
     let selected = TransactionBuilder::new(
         inputs.clone(),
-        outputs.clone(),
+        None,
```

Comment:
This should stay `outputs.clone(),` the test is called `automatic_allotment_provided_in_and_output`

Reply:
Then it fails
```
thread 'client::transaction_builder::basic_outputs::automatic_allotment_provided_in_and_output' panicked at sdk/tests/client/transaction_builder/basic_outputs.rs:2573:6:
called `Result::unwrap()` on an `Err` value: InsufficientAmount { found: 1000000, required: 1014100 }
```
:cry:

Reply:
I changed it because you cannot change the provided outputs, remember

Reply:
But I can add another output to cover the remainder I guess

Reply:
Actually, ignore that, there may be something wrong, I'll investigate

Reply:
The mana calculation needed to factor in the auto allotment in order for the amount requirement to know whether it needed a remainder for mana. It's fixed now.